lib/addons/karpenter/index.ts

```diff
@@ -88,10 +98,10 @@ const RELEASE = 'blueprints-addon-karpenter';
 const defaultProps: HelmAddOnProps = {
     name: KARPENTER,
     namespace: KARPENTER,
-    version: '0.16.3',
+    version: 'v0.20.0',
```

Comment:
Can we increment to 0.22?

Reply:
No objections here. 0.22 deprecates kubernetes 1.20 and older. Will that need to be documented or just referenced?

Reply:
since it is just deprecation (considering 1.20 is already 4 version down) just reference is enough.

Reply:
Addressed.